.github/workflows/release-please.yml

```diff
@@ -0,0 +1,20 @@
+
+# This workflow will perform the following actions when the code is pushed to the main branch.
+# - Trigger release-please action to create a release which needs a test to pass first.
+
+name: release-please-action
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: python
+          package-name: release-please-action
```

Comment:
Shouldn't the package-name be cider-chem or sth?

Reply:
No, it uses the package "release-please-action" to create the release. Nothing has to do with our package.